exercises/practice/yacht/.meta/gen_tests_toml.exs

```diff
@@ -0,0 +1,48 @@
+# Generate tests.toml file from canonical-data.json
```

Comment:
Interesting approach here, we haven't done anything like this yet to generate test files or test toml configurations so I think this might not work with our current set up for v3. Let me take a think what might need to change to accommodate this method.

Reply:
Great, I'll delete this. Thanks!